README.md

```diff
@@ -2,3 +2,24 @@
 
 [![Build Status](https://travis-ci.org/dremio-hub/dremio-hbase-connector.svg?branch=master)](https://travis-ci.org/dremio-hub/dremio-hbase-connector)
 
+## Usage Instructions
+
+* From the project root run `mvn clean install`. The hbase connector jar should now be present
+in the `target` directory.
+
+* Copy the connector jar over to the `<DREMIO_HOME>/jars` folder on all Dremio nodes.
+
+* Stop Dremio.
+
+* Download and copy the following jars to the `<DREMIO_HOME>/jars/3rdparty` folder on all Dremio nodes.
+    * [hbase-client-2.2.0.jar](https://repo1.maven.org/maven2/org/apache/hbase/hbase-client/2.2.0/hbase-client-2.2.0.jar)
+    * [hbase-common-2.2.0.jar](https://repo1.maven.org/maven2/org/apache/hbase/hbase-common/2.2.0/hbase-common-2.2.0.jar)
+    * [hbase-protocol-2.2.0.jar](https://repo1.maven.org/maven2/org/apache/hbase/hbase-protocol/2.2.0/hbase-protocol-2.2.0.jar)
+    * [hbase-protocol-shaded-2.2.0.jar](https://repo1.maven.org/maven2/org/apache/hbase/hbase-protocol-shaded/2.2.0/hbase-protocol-shaded-2.2.0.jar)
+    * [hbase-shaded-miscellaneous-2.2.0.jar](https://repo1.maven.org/maven2/org/apache/hbase/thirdparty/hbase-shaded-miscellaneous/2.2.0/hbase-shaded-miscellaneous-2.2.0.jar)
+    * [hbase-shaded-netty-2.2.0.jar](https://repo1.maven.org/maven2/org/apache/hbase/thirdparty/hbase-shaded-netty/2.2.0/hbase-shaded-netty-2.2.0.jar)
+    * [hbase-shaded-protobuf-2.2.0.jar](https://repo1.maven.org/maven2/org/apache/hbase/thirdparty/hbase-shaded-protobuf/2.2.0/hbase-shaded-protobuf-2.2.0.jar)
+
+* Start Dremio.
```

Comment:
Consider giving the user the command to do this.

Reply:
Done